Tasks/HelmDeployV0/src/helmcommands/helmregistrylogin.ts

```diff
@@ -9,9 +9,26 @@ Signs into ACR helm registry using service principal
 
 export function addArguments(helmCli: helmcli): void {
     helmCli.addArgument("login");
-    const acrEndpoint = tl.getInput("azureSubscriptionEndpointForACR");
-    const user = tl.getEndpointAuthorizationParameter(acrEndpoint, 'serviceprincipalid', true);
-    const password = tl.getEndpointAuthorizationParameter(acrEndpoint, 'serviceprincipalkey', true);
+
+    let isTokenAuth: boolean = false;
+    const token = tl.getVariable("ACR_ACCESS_TOKEN");
```

Comment:
Can we replace getting a variable from env var with a parameter of addArguments() function?

Reply:
Done